### ISSUE TITLE
allegro.pl: breakage

### ISSUE THREAD

**Issue by byt3c** [uBlockOrigin/uAssets#22819]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://allegro.pl/moje-allegro/moje-konto/karty-platnicze?action=list`

### Category

breakage

### Description

While adding payment card or paying using card in allegro.pl there is a script loaded by secure.payu.com. from d13nmjxye4z54k.cloudfront.net which is getting blocked by rule : 	||cloudfront.net/$from=allegro.pl this is preventing user from adding new card or completting 3dsecure authentication.

### Other extensions used

duck duck go privacy and blacklist autoclose

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.56.0
Firefox: 123
filterset (summary):
 network: 125497
 cosmetic: 53827
 scriptlet: 19051
 html: 1752
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 37425-163, now
  ublock-badware: 7807-14, now
  ublock-privacy: 738-0, now
  ublock-unbreak: 2297-0, now
  ublock-quick-fixes: 159-1, now
  easylist: 81990-638, now
  easyprivacy: 50343-869, now
  urlhaus-1: 7490-0, now
  plowe-0: 3779-1, now
  POL-0: 8698-28, now
  POL-2: 1283-12, now
filterset (user): [empty]
trustedset:
 added: [array of 4 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 12954 ms (selfie)
 maxAssetCacheWait: 199 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 6
 network:
  d13nmjxye4z54k.cloudfront.net: 1
  facebook.com: 1
  gemius.pl: 2
  googletagmanager.com: 1
  ngastatic.com: 1
```
</details>

**Comment by garry-ut99** [comment]
The rule which causes the breakage is in "Official Polish Filters" filter list :
https://github.com/MajkiIT/polish-ads-filter/blob/80e99509b071cda8ed8b86a195e07a0fac53b608/polish-adblock-filters/adblock.txt#L4396

- until it's fixed in the filter list, you can put temporarily into "My filters": 
`@@||d13nmjxye4z54k.cloudfront.net^$script,domain=allegro.pl` 
- or if still broken, then can try:
`||cloudfront.net/$domain=allegro.pl,badfilter`

**Comment by krystian3w** [comment]
It's strange that it was only "today" that the card was started broken - they usually work for me at Allegro (by Curve card).

**Comment by byt3c** [comment]
Same for me- last time i paid was around 2 months ago - probably payu changed something pooling script from cloudfront.
